app/javascript/packs/stylesheets/advanced_filter.scss

```diff
@@ -0,0 +1,50 @@
+.advanced-filter-modal-container {
+  // The aggressive z-index ensures that this modal
+  // will be drawn in front of any other modals except the confirm-dialog box
+  background-color: rgba(20, 20, 20, 0.60);
+  z-index: 1999;
+}
+
+.modal-af {
+  width: 1100px !important;
```

Comment:
I know this is just copy-and-pasted, but the `1100px` is wider than a lot of smaller screens. Is there anything preventing us from saying maybe something like `width: 90% !important`?

Reply:
I didn't initially do this, but I believe this is hardcoded for formatting purposes so I would leave it as is

Reply:
I think my concern with this is that for anyone with a screen width less than 1100px, it's impossible for them to scroll/move to see/access the right side of the modal

Reply:
@timwongj the width of this modal has always been hardcoded so it would be an effort to get it to be responsive.  1100px is fairly small and we haven't had any users have issues with it so it is probably okay for now

Reply:
I think this is a small enough CSS change that if I would like to see it resolved here since it should be as simple as what David suggested I believe, and it is certainly a problem.

Reply:
Would it be a big effort? Why not just have it be a percentage width as David suggested above?

Reply:
![Screen Shot 2021-02-02 at 10 36 52 AM](https://user-images.githubusercontent.com/35042815/106623439-9c551280-6542-11eb-9cd1-9edd22e3ddb8.png)

```
.modal-af {
  width: 90% !important;
  max-width: 1100px !important;
}
```

this works fine until you're at small screens.  like I said this is more involved styling change than just changing the width.  given that this PR was about adding tests this should probably be a separate ticket

Reply:
yeah I do agree that it would be a lot of work to make it responsive, so maybe it's something we can just keep an eye out for/think about in the future